src/akamai/js/akamaiHD_flash.js

```diff
@@ -311,6 +311,21 @@ require("../../../html5-common/js/utils/constants.js");
      * @method OoyalaAkamaiHDFlashVideoWrapper#destroy
      */
     this.destroy = function() {
+      // Pause the video
+      this.pause();
+      
+      // Reset the source
+      this.setVideoUrl('');
```

Comment:
This code was added as part of PBF-449 fix in OSMF plugin. So we are doing the similar implementation in Akamai HD also